Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -263,6 +263,34 @@ void OnChildrenCollectionChanged(object sender, NotifyCollectionChangedEventArgs
 			UpdateIgnoreContainerAreas();
 		}
 
+		void TeardownPage(Page page, int i)
+		{
+			page.PropertyChanged -= OnPagePropertyChanged;
+		}
+
+		void SetupPage(Page page, int i)
+		{
+			page.PropertyChanged += OnPagePropertyChanged;
+		}
+
+		void Reset()
+		{
+			var i = 0;
+			foreach (var page in Element.Children)
+				SetupPage(page, i++);
+		}
+
+		private void OnPagePropertyChanged(object sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName == Page.TitleProperty.PropertyName)
+			{
+				var page = (Page)sender;
+				var index = Element.Children.IndexOf(page);
```

Comment:
@jimmgarrido While verifying bug https://bugzilla.xamarin.com/show_bug.cgi?id=43734 using this commit, I met a regression crash when disposing my TabbedPage. The crash occured because OnPagePropertyChanged was called and Element was null. In my work area I made a quick fix with null-checking Element like this:

```
            if (e.PropertyName == Page.TitleProperty.PropertyName)
            {
                if (Element?.Children != null)
                {
                    var page = (Page)sender;
                    var index = Element.Children.IndexOf(page);
                    TabLayout.Tab tab = _tabLayout.GetTabAt(index);
                    tab.SetText(page.Title);
                }
            }
```